redhat-developer/developers.redhat.com
# DEVELOPER-4521 Drupal patch to fix help text not showing for link fields

Added patch to composer.json for a known bug in Drupal affecting the Help Text for link fields that have the Link Text subfield disabled. https://www.drupal.org/node/2421001

### Review Steps

1. Go to Manage Fields for a content type with a link field
2. Click edit on the link field
3. Add some text to the Help Text field
4. Check Disabled on the Allow link text field
5. Save
6. Go to create a new page for that content type
7. The link field should display the help text
